tools/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM python:3.10-slim-bullseye
+MAINTAINER Opentrons Engineering <engineering@opentrons.com>
+
+ENV BUILDENV_ROOT=/build-environment
+ENV POETRY_HOME=/build-environment/poetry
+ENV POETRY_VERSION=1.2.1
+ENV PATH=$PATH:/build-environment/poetry/bin
+ENV POETRY_VIRTUALENVS_IN_PROJECT=true
+ENV HOME=/build-environment
+
+RUN apt update && apt install -y curl wget file git
+RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+RUN mkdir /build-environment
+ADD . /build-environment/tools
+
+# TODO: This should be a release version of the SDK, but one does not
+# exist yet.
+RUN \
+    cd /build-environment \
+    && wget https://opentrons-buildroot-ci.s3.amazonaws.com/439bccec-fff5-4501-9074-f463408da59a/opentrons-buildroot/arm-buildroot-linux-gnueabihf_sdk-buildroot.tar.gz \
```

Comment:
Will this URL change? If so, we should set it as an build arg or env var. 

If the URL does not change, but the underlying content does, using `ADD <url>` would be a good usage here, because if the content does not change it won't break your cache. But if it does change it will redownload it.

See https://github.com/Opentrons/opentrons-emulation/blob/main/docker/Dockerfile#L122-L128 for an example

Reply:
I wonder if ADDing in a separate build stage, building, then using a `COPY --from` to get the right files would be a good idea? 
That way you aren't adding to the image size but you are also getting the caching capability.

Reply:
Another issue to consider, if we use a fixed point address that we are downloading from, if the file underneath updates, say from v1.0 to v2.0. Unless the actual path changes in the Dockerfile, docker will not know that the file has changed and use the old cached version. 

Using ADD gets around this

Reply:
we would not be doing that. like with robot updates, the URL has the version in it. we'll add a build arg to specify the version which will bust the cache.

I think this might be a good time to keep it simple, honestly. The current dockerfile works, mostly caches well, and it'll change infrequently anyway. I think it's good.